packages/server/assets/automate/typedefs/automate.graphql

```diff
@@ -24,7 +24,7 @@ type AutomationRevisionFunction {
 
 type VersionCreatedTriggerDefinition {
   type: AutomateRunTriggerType!
-  model: Model!
+  model: Model
```

Comment:
Why do we need to change these to nullable?

Reply:
@gjedlicska Cause you can delete them